app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DepartureView.java

```diff
@@ -140,13 +145,16 @@ private void updateDepartureTime() {
         // Keep countDown approximately in sync.
         if (mHandler != null) {
             mHandler.postDelayed(() -> {
-                mCountDown--;
                 updateDepartureTime();
```

Comment:
a method reference for a single function lambda would probably be better style

Reply:
good idea, fixed it